### PR TITLE
Fix for static reference rewriting

### PR DESCRIPTION
**Pull request by royteeuwen**
Add possibility to use attribute names that contain a colon for the StaticReferenceRewriteTransformerFactory by providing osgi properties to change the tag - attribute separator

This could be useful for the image tags in svg's:

<image xlink:href="/content/dam...">

### CODE REVIEW

**Comment by royteeuwen** [comment]
Hey @joerghoh , could you please have a look at this one? Not sure if in the release 5.0.0 this one would be the new OSGi DS annotations, but I would be happy to change them to OSGi DS if necessary

**Comment by royteeuwen** [comment]
@badvision  could you review this?

**Comment by badvision** [comment]
If possible can we update to target 5.0 release?   I think this is a great addition but is it something we're needed in an incremental release or can it wait for the next major release?

**Comment by royteeuwen** [comment]
Hey @badvision ,

Thats perfectly fine, we have an internal fork that we use until this is available ;)!

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 - #2386 - Make folder titles overwrite optional for asset ingestor
 - #2432 - AemEnvironmentFilter: use chunked encoding (fixes #2425)
 - #2416 - Fixing workflow package path calculation in WorkflowPackageManager service
+- Add possibility to use attribute names that contain a colon for the StaticReferenceRewriteTransformerFactory
```
Comment (davidjgonzalez) [inline CHANGELOG.md:29]:
```suggestion
- ##2429 - Add ability to use attribute names that contain a colon for the StaticReferenceRewriteTransformerFactory
```

**Review by davidjgonzalez — APPROVED** [review]

**Comment by royteeuwen** [comment]
@davidjgonzalez  same remark here, just tell me where to update it for OSGi DS and I will do it as well ;)

**Pull request merged**